packages/deploy/src/stateTransform.ts

```diff
@@ -87,6 +87,9 @@ function mergeTriggers(stateTriggers, specTriggers): WorkflowState['triggers'] {
             {
               id: crypto.randomUUID(),
               ...pickKeys(specTrigger, ['type']),
```

Comment:
Instead of all that spreading, you could do this:
```
{
  id: crypto.randomUUID(),
  ...pickKeys(specTrigger, ['type', specTrigger.type === 'cron' && 'cron_expression'].filter(Boolean)),
}
```